frontend/ui/elements/common_info_menu_table.lua

```diff
@@ -49,5 +51,32 @@ common_info.report_bug = {
         })
     end
 }
+if Device:isKobo() then
+    common_info.reboot = {
+        text = _("Reboot"),
+    	  callback = function()
```

Comment:
You probably want to set the `_entered_poweroff_stage` flag too here like what's done in `poweroff_action`. Bonus point, create a `reboot_action` function in UIManager and use it here ;)